tests/Unit/Models/QuestionTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Engelsystem\Test\Unit\Models;
+
+use DMS\PHPUnitExtensions\ArraySubset\ArraySubsetAsserts;
+use Engelsystem\Models\User\User;
+use Engelsystem\Test\Unit\HasDatabase;
+use Engelsystem\Test\Unit\TestCase;
+use Engelsystem\Models\Question\Question;
+
+class QuestionTest  extends TestCase
+{
+    use ArraySubsetAsserts;
+    use HasDatabase;
+
+    protected $data = [
```

Comment:
Is useless as a property if its only used once

Reply:
I disagree, this alllows for the test to be further extended without need for refactoring or that the developer does not refactor to this property.

However, if it is something that you would prefer to have moved to the method. I will do that :)

Reply:
Jup, it is ^^